### PR TITLE
change default welcome message text

### PR DESCRIPTION
**Pull request by gunzip**
we must take a decision on how the internal links work with emails

### CODE REVIEW

**Comment by cloudify** [comment]
@gunzip I would say that for now, just keep the link as is, if the user has the app installed, it will open it, or else I guess the browser will just ignore the link

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by gunzip** [comment]
needs review

File: lib/profile_events_queue_handler.ts [hunk 1]
```diff
@@ -49,33 +49,48 @@ const publicApiKey = getRequiredStringEnv("PUBLIC_API_KEY");
...
+          // tslint:disable-next-line:restrict-plus-operands
+          profile.email ? ":" + profile.email : ""
+        }.
+Se hai già usato pagoPA per effettuare pagamenti verso la Pubblica Amministrazione, potrai vedere lo storico delle transazioni
+ed eventuali carte di credito salvate nella sezione [portafoglio](ioit://WALLET_HOME).
+Altrimenti, sempre dal [portafoglio](ioit://WALLET_HOME) puoi aggiungere i tuoi metodi di pagamento preferiti, oppure pagare direttamente
+un avviso pagoPA leggendo il QR code di un avviso cartaceo.
+Se qualcosa non ti dovesse essere chiaro durante l'utilizzo dell'app, clicca il punto di domanda che trovi in alto a destra.`,
+
+        subject: `Benvenuto ! Scopri le funzioni principali e impara a usare l'app dei servizi pubblici.`
```
Comment (cloudify) [inline lib/profile_events_queue_handler.ts]: @matteodesanti  subject looks a little long, also it looks like there's an extra space after _Benvenuto_.

What about: _Benvenuto su IO, l'app dei servizi pubblici!_

File: lib/profile_events_queue_handler.ts [hunk 2]
```diff
@@ -49,33 +49,48 @@ const publicApiKey = getRequiredStringEnv("PUBLIC_API_KEY");
...
+  (_: ExtendedProfile) =>
     NewMessage.decode({
       content: {
-        markdown: `# Hello new user ${profile.email || ""}
+        markdown: `**Scopri come attivare o disattivare i servizi delle Pubbliche Amministrazioni.**
 
-  We welcome you to the Digital Citizenship API program  
-  This is a welcome message to test if the system works.`,
+Scegli i servizi delle Pubbliche Amministrazioni con cui vuoi interagire nell'app.
+Al lancio dell'applicazione la lista dei servizi disponibili comprende tutte quelle a livello nazionale
```
Comment (cloudify) [inline lib/profile_events_queue_handler.ts]: servizi e' maschile: a lista dei servizi disponibili comprende tutti quelli

File: lib/profile_events_queue_handler.ts [hunk 3]
```diff
@@ -49,33 +49,48 @@ const publicApiKey = getRequiredStringEnv("PUBLIC_API_KEY");
...
+(ad esempio l'ACI) e quelle relative ad alcune regioni e ad alcuni comuni, ma non temere:
```
Comment (cloudify) [inline lib/profile_events_queue_handler.ts]: l'ACI -> ACI ?

File: lib/profile_events_queue_handler.ts [hunk 4]
```diff
@@ -49,33 +49,48 @@ const publicApiKey = getRequiredStringEnv("PUBLIC_API_KEY");
...
+ti scriveranno solo i servizi che hanno qualcosa di specifico da dire proprio a te! ;-)
+Puoi disattivare eventuali servizi a cui non sei interessato oppure scegliere su quale canale
```
Comment (cloudify) [inline lib/profile_events_queue_handler.ts]: scritto cosi' appare tutto su una riga, magari serve una riga vuota

File: lib/profile_events_queue_handler.ts (hunk 2 above)
Comment (gunzip) [reply lib/profile_events_queue_handler.ts]: credo si riferisca alle amministrazioni

**Pull request merged**